DataFormats/TrackReco/src/HitPattern.cc

```diff
@@ -1013,3 +1026,17 @@ bool HitPattern::insertExpectedOuterHit(const uint16_t pattern) {
 
   return true;
 }
+
+Run3ScoutingHitPatternPOD HitPattern::run3ScoutingHitPatternPOD() const {
+  Run3ScoutingHitPatternPOD run3ScoutingHitPatternPOD_;
```

Comment:
Would avoid the final `_` (usually used for class members).

Reply:
Done.